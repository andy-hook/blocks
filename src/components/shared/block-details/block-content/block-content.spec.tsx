import React from "react"
import BlockContent from "./block-content"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<BlockContent />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <BlockContent />)
    expect(tree).toMatchSnapshot()
  })
})
