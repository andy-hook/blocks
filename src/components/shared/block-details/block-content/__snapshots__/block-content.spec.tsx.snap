// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockContent /> renders correctly 1`] = `
<article
  className="sc-gZMcBi kxLsOT"
>
  <header
    className="sc-htoDjs bzhYRm"
  >
    <div
      className="sc-iwsKbI ldERec"
    >
      <div
        className="sc-htpNat bHaBzt sc-dnqmqq ctRtNT"
        size="lg"
      >
        #Â 
      </div>
      <div
        className="sc-bdVaJa jfEAsh"
        size="lg"
      >
         Transactions
      </div>
    </div>
    <a
      className="sc-bZQynM fJEiEe"
      href="https://www.google.com"
      type="primary"
    >
      <span
        className="sc-gzVnrw feWhnB"
      >
        View transactions
      </span>
    </a>
  </header>
  <ul
    className="sc-gqjmRU hnZXkc"
  >
    <div
      className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
    >
      <div
        className="sc-htpNat hwnZXv"
        size="sm"
      >
        Size
      </div>
      <div
        className="sc-EHOje foZbCf"
      >
        <div
          className="sc-bdVaJa htfwrO"
          size="lg"
        />
      </div>
    </div>
    <div
      className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
    >
      <div
        className="sc-htpNat hwnZXv"
        size="sm"
      >
        Difficulty
      </div>
      <div
        className="sc-EHOje foZbCf"
      >
        <div
          className="sc-bdVaJa htfwrO"
          size="lg"
        />
      </div>
    </div>
    <div
      className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
    >
      <div
        className="sc-htpNat hwnZXv"
        size="sm"
      >
        Total Difficulty
      </div>
      <div
        className="sc-EHOje foZbCf"
      >
        <div
          className="sc-bdVaJa htfwrO"
          size="lg"
        />
      </div>
    </div>
    <div
      className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
    >
      <div
        className="sc-htpNat hwnZXv"
        size="sm"
      >
        Gas Used
      </div>
      <div
        className="sc-EHOje foZbCf"
      >
        <div
          className="sc-bdVaJa htfwrO"
          size="lg"
        />
      </div>
    </div>
    <div
      className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
    >
      <div
        className="sc-htpNat hwnZXv"
        size="sm"
      >
        Gas Limit
      </div>
      <div
        className="sc-EHOje foZbCf"
      >
        <div
          className="sc-bdVaJa htfwrO"
          size="lg"
        />
      </div>
    </div>
  </ul>
</article>
`;
